projects/aws/eks-anywhere/build/fetch_binaries.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
For now, we can use [this](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/fluxcd/kustomize-controller/build/fetch_binaries.sh) script and directly copy over `kubectl` into the image like [this](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/fluxcd/kustomize-controller/docker/linux/Dockerfile#L9)

Reply:
done